Tests/MapboxMapsTests/MapView/Integration Tests/FeatureQueryingTest.swift

```diff
@@ -78,8 +78,9 @@ internal class FeatureQueryingTest: MapViewIntegrationTestCase {
                     if case .success(let unfilteredFeatures) = unfilteredFeatures,
                        case .success(let filteredFeatures) = filteredFeatures {
 
-                        let expectedFilteredFeatures = unfilteredFeatures.filter { feature in
-                            return feature.geometry.type == .Point
+                        let expectedFilteredFeatures = unfilteredFeatures.filter { queriedFeature in
+//                            return queriedFeature.feature.geometry.geometryType == MBXGeometryType.point
+                            return queriedFeature.feature.geometry.geometryType == MBXGeometryType(1)
```

Comment:
`MBXGeometryType(1)` is a bit confusing - what's going in here? Can we lose the commented line above, and replace with a descriptive comment?

Reply:
sure thing